docs/supported_instruments.rst

```diff
@@ -97,6 +97,14 @@ DE2 VEFI
 .. automodule:: pysatNASA.instruments.de2_vefi
    :members:
 
+.. _de2_vefimag:
+
+DE2 VEFIMAGB
```

Comment:
Is the magnetometer part of the VEFI package? Seems like putting this dataset under the 'mag' tag is more in line with the organizational structure, including the C/NOFS VEFI support, rather than as its own instrument.